sympy/geometry/ellipse.py

```diff
@@ -585,6 +612,7 @@ def intersection(self, o):
 
         Notes
         -----
+
```

Comment:
Also revert this c0106cb9743b6771a3e7a8b831d9c4428f84611c, if you don't have to touch other work.